src/components/AppButton.vue

```diff
@@ -40,6 +40,11 @@ defineProps({
     type: String,
     default: '_blank',
   },
+  size: {
+    type: String,
+    default: 'md',
+    validator: value => ['sm', 'md'].includes(value),
```

Comment:
Please explain why this should be lg. It's counterintuitive to me that 'lg' is supposed to be the default value instead of md

Reply:
Please ignore the above comment. I wasn't aware that other button sizes got removed. I'm applying the default size to all buttons since it's the only one we use in the app